mobile/lib/src/domain/eeg_data/eeg_data.dart

```diff
@@ -0,0 +1,14 @@
+// EEGData can be extended later to add our metrics
+class EEGData {
+  String id;
+  List<List> _values;
```

Comment:
why not `List<List<string>>`?

Reply:
Les values contiennent un header de string pour le format de lecture openBCI mais les données eeg sont des ints. Je peux convertir de int en string pis faire une liste de string, mais c'est pas vraiment nécessaire. Penses tu que c'est mieux d'avoir une liste de int et de simplement ajouter le header avant de save to csv ?